### PR TITLE
Fix issues

### PR DESCRIPTION
**Pull request by mikeyrayvon**

### CODE REVIEW

File: partials/archive-exhibitor/introduction.php [hunk 1]
```diff
@@ -30,9 +30,18 @@
       <div class="row">
 <?php 
   if (!empty($exhibitors_page_image_id)) {
+    $attachment = get_post( $exhibitors_page_image_id );
```
Comment (varffs) [inline partials/archive-exhibitor/introduction.php:33]: this is duplicated code so could be done as a partial

Comment (varffs) [inline partials/archive-exhibitor/introduction.php:33]: or render function

**Comment by varffs** [comment]
I think issues should be fixed with their own PRs as grouping blocks progress. I would merge this but have queries for 2 of 5 issue fixes so I can't. It is also difficult to cross reference everything as you have to go back to the issues to find what the fix is for

File: partials/archive-exhibitor/introduction.php (hunk 1 above)
Comment (mikeyrayvon) [inline partials/archive-exhibitor/introduction.php:33]: im making them all into a partial in a later branch

**Comment by mikeyrayvon** [comment]
can you merge it now?

**Pull request merged**